test/unit/config_report_status_calculator_test.rb

```diff
@@ -1,5 +1,5 @@
 require 'test_helper'
-class ReportStatusCalculatorTest < ActiveSupport::TestCase
+class ConfigReportStatusCalculatorTest < ActiveSupport::TestCase
```

Comment:
Probably a mistake during rebasing.
Renaming back to `ConfigReportStatusCalculator`